src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,83 @@
+package duke;
+
+import java.util.ArrayList;
+
+import duke.exception.DukeException;
+import duke.exception.DuplicateTaskException;
+import duke.exception.WrongUsageException;
+import duke.task.Task;
+
+/**
+ * Class for a task list were we store all the tasks
+ */
+public class TaskList {
+    protected ArrayList<Task> taskArrayList;
```

Comment:
I like that you explicitly added the word `array` in the variable name to distinguish it from `TaskList`. This naming practice makes it super clear.